modules/op-resolver-hub-config.adoc

```diff
@@ -33,9 +33,20 @@ spec:
       default-artifact-hub-pipeline-catalog: tekton-catalog-pipelines # <3>
       defailt-kind: pipeline # <4>
       default-type: tekton # <5>
+      tekton-hub-api: "https://my-custom-tekton-hub.example.com" # <6>
+      artifact-hub-api: "https://my-custom-artifact-hub.example.com" # <7>
 ----
 <1> The default {tekton-hub} catalog for pulling a resource.
 <2> The default {artifact-hub} catalog for pulling a task resource.
 <3> The default {artifact-hub} catalog for pulling a pipeline resource.
 <4> The default object kind for references.
 <5> The default hub for pulling a resource, either `artifact` for {artifact-hub} or `tekton` for {tekton-hub}.
+<6> The {tekton-hub} API used, if the `default-type` option is set to `tekton`.
+<7> The {artifact-hub} API used, if the `default-type` option is set to `artifact`.
++
+[IMPORTANT]
+====
+If you set the `default-type` option to `tekton`, you must configure your own instance of the {tekton-hub} by setting the `tekton-hub-api` value.
```

Comment:
Is this actually true? Isn't there a default Tekton Hub?

Reply:
Hello @jkandasa can you please confirm the above? Thank you!

Reply:
@ekristova There is no public(or default) tekton hub. So we must to set it manually.
Source: https://github.com/tektoncd/pipeline/blob/main/docs/hub-resolver.md#configuring-the-hub-api-endpoint
```
When setting the type field to tekton, you must configure your own instance of the Tekton Hub by setting the TEKTON_HUB_API environment variable...
```